packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -442,14 +426,14 @@ export class SharedTreeBranch<
 
 	/**
 	 * Spawn a new branch that is based off of the current state of this branch.
-	 * Changes made to the new branch will not be applied to this branch until the new branch is merged back in.
-	 *
-	 * @remarks Forks created during a transaction will be disposed when the transaction ends.
+	 * @param commit - the commit to base the new branch off of. Defaults to the head of this branch.
+	 * @remarks Changes made to the new branch will not be applied to this branch until the new branch is merged back in.
```

Comment:
Nit: should we link to the `merge` method here for convenience?

Reply:
Sure!